src/main/java/com/eternalcode/discordapp/experience/listener/ExperienceReactionListener.java

```diff
@@ -0,0 +1,32 @@
+package com.eternalcode.discordapp.experience.listener;
+
+import com.eternalcode.discordapp.experience.ExperienceConfig;
+import com.eternalcode.discordapp.experience.ExperienceRepository;
+import com.eternalcode.discordapp.experience.ExperienceService;
+import net.dv8tion.jda.api.events.message.react.MessageReactionAddEvent;
+import net.dv8tion.jda.api.events.message.react.MessageReactionRemoveEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+public class ExperienceReactionListener extends ListenerAdapter {
+
+    private final ExperienceRepository experienceRepository;
+    private final ExperienceConfig experienceConfig;
+    private final ExperienceService experienceService;
+
+    public ExperienceReactionListener(ExperienceRepository experienceRepository, ExperienceConfig experienceConfig) {
+        this.experienceRepository = experienceRepository;
+        this.experienceConfig = experienceConfig;
+        this.experienceService = new ExperienceService(this.experienceRepository);
```

Comment:
To samo co wcześniej, w dodatku duplikujesz ten service per Listener nie potrzebnie